smoke-tests-otel-starter/spring-boot-3/src/test/java/io/opentelemetry/smoketest/GraalVmNativeMongodbSpringStarterSmokeTest.java

```diff
@@ -0,0 +1,14 @@
+package io.opentelemetry.smoketest;
+
+import org.junit.jupiter.api.condition.EnabledInNativeImage;
+
+/**
+ * GraalVM native image doesn't support Testcontainers, so the docker container is started manually
```

Comment:
Perhaps explained that it does not work with MongoDB used with Testcontainers and GraalVM native.

In other cases, it seems that Testcontainers can work well with GraalVM native: https://info.michael-simons.eu/2023/10/25/run-your-integration-tests-against-testcontainers-with-graalvm-native-image/

Reply:
Interesting! I'll change the comment to avoid confusion.